lib/mysqlDaoQueryHelper.js

```diff
@@ -261,73 +262,96 @@ class MysqlDaoQueryHelper {
 
     _deleteBase(whereClause) {
         let sqlObject =
-            Squel.delete({parameterCharacter: '!!@@##$$%%'}) // See comment in _getBase for explanation of paramChar
+            this._squel.delete({parameterCharacter: '!!@@##$$%%'}) // See comment in _getBase for explanation of paramChar
                 .from(this._tableName);
 
         sqlObject = this._appendWhereClause(sqlObject, whereClause);
         return sqlObject;
     }
 
     /**
-     * @param {object} obj
-     * @param {object} [config]
-     * @param {boolean} [config.dontCleanMysqlFunctions] - defaults to true; includes CURRENT_TIMESTAMP and NOW();
-     *
-     * @returns {object} - cleans the values and returns an object with snake case props --> clean values
+     * @private
+     * @param {Object|Object[]} obj
+     * @param {Boolean} [dontCleanMysqlFunctions] whether or not to allow MySQL functions to exist in values
+     * @returns {Object} - cleans the values and returns an object with snake_case props --> clean values
      */
-    _cleanAndMapValues(obj, config) {
-        const defaultConfig = {
-            dontCleanMysqlFunctions: true
-        };
-
-        config = _.merge(defaultConfig, config);
+    _cleanAndMapValues(obj, dontCleanMysqlFunctions = true) {
+        if (Array.isArray(obj)) {
+            const propertyMappings = MysqlDaoQueryHelper._getPropertyMappings(objs[0]);
```

Comment:
Ran the tests locally and got an error here.  `objs` isn't defined.  I'm going to update locally before I push anything to master.